src/l500/ac-trigger.cpp

```diff
@@ -1297,38 +1297,38 @@ namespace ivcam2 {
         // Algo was written with specific receiver gain (APD) in mind, depending on
         // the FW preset (ambient light)
         auto & depth_sensor = _dev.get_depth_sensor();
-        auto & ambient_light = depth_sensor.get_option( RS2_OPTION_AMBIENT_LIGHT );
-        float raw_ambient = ambient_light.query();
+        auto & digital_gain = depth_sensor.get_option(RS2_OPTION_DIGITAL_GAIN);
+        float raw_digital_gain = digital_gain.query();
         auto & apd = depth_sensor.get_option( RS2_OPTION_AVALANCHE_PHOTO_DIODE );
         float raw_apd = apd.query();
         _receiver_gain = int( raw_apd );
-        _ambient = ( rs2_ambient_light ) int( raw_ambient );
-        switch( _ambient )
+        _digital_gain = ( rs2_digital_gain ) int(raw_digital_gain);
+        switch(_digital_gain)
         {
-        case RS2_AMBIENT_LIGHT_LOW_AMBIENT:  // SHORT
+        case RS2_DIGITAL_GAIN_LOW_GAIN:
             if( _receiver_gain != 18 )
             {
                 if( ! invalid_reason.empty() )
                     invalid_reason += ", ";
                 invalid_reason += to_string()
-                               << "low-ambient (SHORT) receiver gain (" << raw_apd << ") != 18";
+                               << "low digital gain - receiver gain (" << raw_apd << ") != 18";
```

Comment:
Keep the FW preset please.
This is VERY confusing terminology.
How about `receiver gain (X) of 18 is expected with low digital gain (SHORT)`?